docs/ref/contrib/admin/index.txt

```diff
@@ -59,7 +59,7 @@ visiting the URL you hooked it into (``/admin/``, by default).
 
 If you need to create a user to login with, use the :djadmin:`createsuperuser`
 command. By default, logging in to the admin requires that the user has the
-:attr:`~.User.is_superuser` or :attr:`~.User.is_staff` attribute set to
+:attr:`~.User.is_superuser` and :attr:`~.User.is_staff` attribute set to
```

Comment:
With this change, wouldn't this imply that `is_superuser` is required in all cases?

Reply:
I agree with Jacob, we should chop ``:attr:`~.User.is_superuser` or ``
```suggestion
:attr:`~.User.is_staff` attribute set to
```

Reply:
It's a regression in 939dcff24f8e97d114595b102fb12348da482135.

Reply:
Yes, my proposed change is not entirely correct, but it might be enough to serve as an introduction, while more detail is given later on.

With only is_staff you can login, but change anything without further permissions.

Reply:
`is_superuser` is not required for logging in to the admin, so it's confusing. IMO 

_"If you need to create a user to login with, use the ``createsuperuser`` command. By default, logging in to the admin requires that the user has the ``is_staff`` attribute set to ``True``."_

works fine.

Reply:
Question: Should we change the code to match the documentation? Shouldn't a superuser have access to everything, as they do for permissions?

So `is_staff` ultimately could have been a permission for "can access admin site" if global permissions were a thing. (I know there's an open ticket for that still.) But because they aren't we have a boolean that is separate to the machinery that allows a superuser to bypass all permissions checks.

I guess the only issue is that there may be other places where `is_staff` is used, e.g. `@staff_member_required`, that also don't use `is_superuser`.

Just thought I'd ask…

Reply:
This is a small documentation fix, that can be backported. Changes in the current behavior (if any) should be handled separately.

Reply:
Oh, I get that. It just seems arbitrary whether we fix the documentation to match the code or vice versa sometimes.

And I'm happy with this approach.

Reply:
@felixxm I've accepted your suggestion and added an additional commit to put the remainder on the same line.